zipkin-thrift/build.gradle

```diff
@@ -0,0 +1,11 @@
+buildscript {
+	repositories {
+		jcenter()
+	}
+
+	dependencies {
+		classpath griddle
+	}
+}
+
+apply plugin: 'idl'
```

Comment:
No, I was more confused because most Gralde plugins that I've seen define one plugin per dependency/jar